### PR TITLE
Refactor to use gql queries

### PR DESCRIPTION
**Pull request by Afonso-2403**
Make use of the gql library feature to build queries from the schema definition, instead of using raw strings

### CODE REVIEW

File: src/isar_exr/api/energy_robotics_api.py [hunk 1]
```diff
@@ -20,56 +22,86 @@
...
+from gql.dsl import DSLVariableDefinitions, DSLMutation, DSLQuery, dsl_gql
+
+
+def to_dict(obj):
+    return json.loads(json.dumps(obj, default=lambda o: o.__dict__))
 
 
 class EnergyRoboticsApi:
     def __init__(self):
         self.client = GraphqlClient()
```
Comment (Christdej) [inline src/isar_exr/api/energy_robotics_api.py:34]: Would consider adding 
`self.schema = self.client.schema`
for easier accessibility of schema

File: src/isar_exr/api/energy_robotics_api.py [hunk 2]
```diff
@@ -317,164 +289,206 @@ def wake_up_robot(
                     f"Not able to wake up robot after '{timeout}' seconds."
                 )
             sleep(1)
+        print("success waking up robot")
```
Comment (Christdej) [inline src/isar_exr/api/energy_robotics_api.py:292]: This should be part of a logger rather than a print

**Review by Christdej — CHANGES_REQUESTED** [review]

File: src/isar_exr/api/energy_robotics_api.py (hunk 2 above)
Comment (Afonso-2403) [reply src/isar_exr/api/energy_robotics_api.py]: Was from local testing, had forgotten to remove it

**Review by Christdej — APPROVED** [review]
LGTM

**Pull request merged**